src/JabbR-Core/wwwroot/Chat.js

```diff
@@ -7,6 +7,9 @@
 (function ($, connection, window, ui, utility) {
     "use strict";
 
+    // Comment out to ensure logging is turned off.
```

Comment:
Remove the dev debugging here, we can always set in browser. This way we don't have to remember to remove it later (dont want debug on by default in end user browsers)